python/__init__.py

```diff
@@ -11,6 +10,7 @@
 #
 # For information on the complete list of contributors to the
 # XCFun library, see: <https://xcfun.readthedocs.io/>
-#
+
+# -*- coding: utf-8 -*-
```

Comment:
Can you move this to the first line? Python 2 does not recognize it otherwise (I know, end of life etc.) - for Python 3 it is not needed anymore because utf-8 is the default, but it should still be on the first line ...

Reply:
Done.